HISTORY.md

```diff
@@ -40,7 +40,7 @@ That's far from the only user-visible change though! The entire list of improvem
 
 * The "deploy/example_catalog" directory has gained a new very simple example app (vanilla CentOS 7) and a new complicated one (CDH 5.14.2 with Pig, Hive, and Hue support).
 
-* The "configcli" tool used in app setup is now included in this repo, in the "nodeprep" directory.
+* The "configcli" tool used in app setup is a separate git repository at https://github.com/bluek8s/configcli
```

Comment:
This is the history of particular released versions, so no revisionism. :-)

Reply:
If you want, you can start a new v0.3.0 section at the top of the file (with a ??? release date) that incudes moving configcli as a bullet point feature.

Rebase to the current state of master branch first tho.